event-checklist.md

```diff
@@ -19,7 +19,7 @@ The steps can be summarised as;
 * [ ] Read through the [event subcommittee policy](subcommittee_policy_v2.md)
 * [ ] Ensure all members of your team are [members of Linux Australia](https://www.linux.org.au/membership)
 * [ ] Request a new subcommittee be created by [emailing the Council](mailto:council@linux.org.au). An example of what should be included in the request is given below.
-* [ ] Understand expectations from Linux Australia that the event is expected to make a profit of $AUD 11 per attendee, or as negotiated, and that all profits from the event are returned to Linux Australia to be re-invested in the region's Linux and open source community.
+* [ ] Understand expectations from Linux Australia that the event is expected to make a minimum profit to cover organisational expenses such as banking, auditing, insurance and so on, and that all profits above and beyond this amount from the event are returned to Linux Australia to be re-invested in the region's Linux and open source community.

```

Comment:
So I'm totally nit-picking, but I wouldn't call the $11 thingy a profit. It should be accounted for as an expense.

I'd also be careful with the note about the profit being re-invested. This is true, but it's difficult to point to $event's profit paid for $project. It'd be easier to note that the profit is returned to linux australia to be used within their values etc.

It might also be helpful to have a check item to discuss an expected profit with LA (as per the subcommittee policy). For example, we ask LCA to aim for a certain amount, and I think pyconau is heading that way a little too.